### ISSUE TITLE
Content protection

### ISSUE THREAD

**Issue by jronallo** [IIIF/iiif-av#36]
# Description

Content protection. View the content but don't need to authenticate.  Mostly for IPR issues. HTML5 video player, solution was to use a unique token every time. Play content once with the token. Can go forwards and backwards. But on a download manager you can't get it. Stronger than Image API protection.  Would need to webscrape javascript.
Need to pass token to the streaming environment. Could be in URL params, cookie, header etc.

## Proposed Solutions

Tags: download
IIIF: Auth API

Issue: Question as to whether a redirecting video/audio element src redirects might cause problems?  Otherwise can have a IIIF implementation of the Auth API, either as a shim on top of existing system or as a new system.

Auth API looks like it should work with HTML5 video/audio as long as resource is on the same domain as auth server (cookies), but still allows downloading the local file from the browser once authenticated.

Solution: Specify token service at level 3?
MPEG-DASH and HLS can make it hard to download.

IIIF Tile Harvester in Python:
   https://github.com/azaroth42/iiif-harvester/blob/master/tilescraper.py

## Additional Background

Source: [BL workshop notes](https://docs.google.com/document/d/1jl4JEfPj8--YHW0HgmkEyTrTcmJI42jFyK9vYV7r6bo/edit)
Interest: 60%

**Comment by jronallo** [comment]
Does Encrypted Media Extensions along with DRM solve this issue? 

There are similar tools to the tile harvester for recombining HLS. While HLS makes it more difficult for many to do this it does not alone make it sufficiently difficult for some cases. What is segmented can be recombined. I think of HLS as more appropriate for live streaming and adaptive streaming cases than for real content protection.

**Comment by azaroth42** [comment]
I would propose leaving this entire topic alone until the mess with EME plays itself out one way or the other.

**Comment by jronallo** [comment]
I agree with waiting. Is a label like "defer" something we could add to note a decision on that? Is this a decision that would need the editors to sign off on? Just wondering about process for making it clear that this is something that we'd wait on.

**Comment by azaroth42** [comment]
Should confirm in the WG that there's agreement on `defer`, and if so we can tag it and come back to it later.
